src/game/server/instagib/round_stats.cpp

```diff
@@ -123,41 +123,53 @@ void IGameController::GetRoundEndStatsStrPsv(char *pBuf, size_t Size)
 	int ScoreLimit = m_GameInfo.m_ScoreLimit;
 	int TimeLimit = m_GameInfo.m_TimeLimit;
 
-	std::optional<std::string> RedClan = std::nullopt;
-	std::optional<std::string> BlueClan = std::nullopt;
+	const char *RedClan = nullptr;
```

Comment:
Can you please call it `pRedClan` not `RedClan` because it is a pointer.

Reply:
omagawd sowwy🥺🥺🥺🥺